mico-core/src/test/java/io/github/ust/mico/core/ServiceControllerTests.java

```diff
@@ -786,6 +790,40 @@ public void getDependees() throws Exception {
                 .andReturn();
     }
 
+    @Test
+    public void deleteServiceWithDeployedService() throws Exception {
+        MicoService service = new MicoService()
+            .setShortName(SHORT_NAME)
+            .setVersion(VERSION)
+            .setDescription(DESCRIPTION);
+
+        given(serviceRepository.findByShortName(SHORT_NAME)).willReturn(Collections.singletonList(service));
+        given(micoKubernetesClient.isServiceDeployed(any())).willReturn(true);
+
+        mvc.perform(delete(SERVICES_PATH + "/" + SHORT_NAME)
+            .contentType(MediaTypes.HAL_JSON_UTF8_VALUE))
+            .andDo(print())
+            .andExpect(status().isConflict())
+            .andExpect(status().reason("Service is currently deployed!"));
```

Comment:
Checking the status `.andExpect(status().isConflict())` should be enough, right? Also checking the string of the error reduces the maintenance.